homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -219,15 +219,16 @@ def _async_get_options_flow(
             config_entry: config_entries.ConfigEntry,
         ) -> config_entries.OptionsFlow:
             """Get the options flow for this handler."""
-            if cls.options_flow is None:
```

Comment:
I suppose this case should never happen in real life?

Reply:
It is guarded below (the function is only assigned `if options_flow is not None`)